File: android/app/src/main/java/jagdx/D3DPRESENT_PARAMETERS.java
package jagdx;


import voidawt.Canvas;

public final class D3DPRESENT_PARAMETERS {

    public boolean Windowed;

    public int Flags;

    public int FullScreen_RefreshRateInHz;

    public boolean EnableAutoDepthStencil;

    public int BackBufferHeight = 0;

    public int BackBufferWidth = 0;

    public int PresentationInterval = 0;

    public final int BackBufferCount = 0;

    public int MultiSampleQuality = 0;

    public int BackBufferFormat = 0;

    public int MultiSampleType = 0;

    public final int SwapEffect = 1;

    public int AutoDepthStencilFormat = 0;

    public final Canvas DeviceWindow;

    public D3DPRESENT_PARAMETERS(Canvas arg0) {
        this.DeviceWindow = arg0;
    }
}

File: src/jvmMain/kotlin/jagdx/D3DPRESENT_PARAMETERS.kt
package jagdx

import java.awt.Canvas


class D3DPRESENT_PARAMETERS(private val DeviceWindow: Canvas?) {

	var Windowed: Boolean = false

    private val Flags = 0

    private val FullScreen_RefreshRateInHz = 0


	var EnableAutoDepthStencil: Boolean = false

    private val SwapEffect = 1


	var BackBufferFormat: Int = 0


	var MultiSampleQuality: Int = 0


	var MultiSampleType: Int = 0


	var AutoDepthStencilFormat: Int = 0


	var BackBufferHeight: Int = 0


	var PresentationInterval: Int = 0

    private val BackBufferCount = 0


	var BackBufferWidth: Int = 0
}
